api/src/components/sqs/handler.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+export const generatePDFs = async (event): Promise<object> => {
+    const promises = event.Records.forEach(async (record) => {
+        const { body } = record;
+
+        try {
+            await axios.get(`https://${process.env.STAGE}.api.octopus.ac/v1/publications/${body}/pdf`);
+        } catch (err) {
```

Comment:
async await doesn't work inside forEach loop. This catch block won't catch any axios errors because all the http requests are fired immediately and not awaiting for the actual response.